decidim-admin/app/forms/decidim/admin/static_page_form.rb

```diff
@@ -14,10 +14,16 @@ class StaticPageForm < Form
 
       validates :slug, presence: true
       validates :title, :content, translatable_presence: true
+      validates :slug, format: { with: /^[a-z0-9-]+$/, multiline: true }
+
       validate :slug, :slug_uniqueness
 
       alias organization current_organization
 
+      def slug
+        super.to_s.downcase
```

Comment:
this should be a `before_validation` hook, as this could potentially cause issues when using `find` (the database will be inconsistent).